.gitignore

```diff
@@ -36,3 +36,7 @@ env-config.sh
 /calypso-strings.php
 cover.html
 .test.log
+
+.caches/*
+!.caches/.gitkeep
```

Comment:
Should it just be `.cache`? `.cache/babel/*` for the files maybe?

Guess we could always make it more structured if the need arises.